src/Button/Button.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
```

Comment:
Let's keep the folder hierarchy flat for now, which will help with imports and merging – I don't see much value in putting a module named `Button` inside a folder of the same name

Reply:
We can make it flat for now, but I think we will have to introduce some hierarchy later on in imo.

Reply:
Maybe, but we have no idea what that hierarchy should be yet – I'd say generally hold off on organizing things into folders until it becomes a development problem